services/wfm/src/main/java/org/openkilda/wfm/topology/connecteddevices/service/PacketService.java

```diff
@@ -35,18 +40,23 @@
 public class PacketService {
     private TransactionManager transactionManager;
     private FlowCookieRepository flowCookieRepository;
+    private SwitchRepository switchRepository;
     private ConnectedDeviceRepository connectedDeviceRepository;
+    private SwitchConnectedDeviceRepository switchConnectedDeviceRepository;
 
     public PacketService(PersistenceManager persistenceManager) {
         transactionManager = persistenceManager.getTransactionManager();
         flowCookieRepository = persistenceManager.getRepositoryFactory().createFlowCookieRepository();
+        switchRepository = persistenceManager.getRepositoryFactory().createSwitchRepository();
         connectedDeviceRepository = persistenceManager.getRepositoryFactory().createConnectedDeviceRepository();
+        switchConnectedDeviceRepository = persistenceManager.getRepositoryFactory()
+                .createSwitchConnectedDeviceRepository();
     }
 
     /**
      * Handle LLDP info data.
      */
-    public void handleLldpData(LldpInfoData data) {
+    public void handleFlowLldpData(LldpInfoData data) {
```

Comment:
Why is infodata class not called FlowLldpInfoData? To be consistent with SwitchLldpInfoData.

Reply:
LldpInfoData class will be removed in next patch anyway